Tools/build/generate_sbom.py

```diff
@@ -50,7 +50,7 @@ class PackageFiles(typing.NamedTuple):
         include=["Modules/expat/**"]
     ),
     "pip": PackageFiles(
-        include=["Lib/ensurepip/_bundled/pip-23.3.1-py3-none-any.whl"]
+        include=["Lib/ensurepip/_bundled/pip-23.3.2-py3-none-any.whl"]
```

Comment:
```suggestion
        include=["Lib/ensurepip/_bundled/pip-*-py3-none-any.whl"]
```

See my comment (you made this change while I was writing it 😉)

Reply:
That seems to work.